File: Java/olamundo/src/main/java/br/com/iteris/universidade/olamundo/domain/dto/AvaliacaoCreateRequest.java
package br.com.iteris.universidade.olamundo.domain.dto;

import lombok.Data;

import javax.validation.constraints.NotNull;

@Data
public class AvaliacaoCreateRequest {
    @NotNull(message = "Id do Album deve ser definido")
    private Integer IdAlbum;

    @NotNull(message = "Nota deve ser definido")
    private Integer Nota;

    private String Comentario;
}

File: C#/PrimeiraWebAPI/PrimeiraWebAPI/Domain/DTO/AvaliacaoCreateRequest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace PrimeiraWebAPI.Domain.DTO
{
	public class AvaliacaoCreateRequest
	{
		[Required]
		public int IdAlbum { get; set; }
		[Required]
		public int Nota { get; set; }
		public string Comentario { get; set; }
	}
}
